lib/modules/manager/woodpecker/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ProgrammingLanguage } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+import { extractPackageFile } from './extract';
+
+export const language = ProgrammingLanguage.Docker;
```

Comment:
does woodpecker support other kinds of dependencies beside docker images? are there other types planned?

Reply:
No, currently only Docker Images are supproted. Another thing would be to use `apt`, `apk` or `dnf` ... in a comment with a pinned package version, but I think that is out of this project's scope and not really doable.